quinn/src/recv_stream.rs

```diff
@@ -506,6 +554,15 @@ impl From<ReadableError> for ReadError {
     }
 }
 
+impl From<ResetError> for ReadError {
```

Comment:
What's the motivation for this impl?

Reply:
This would work nicely for the apps that want to either `read` or wait for `reset` to unify the error type without adding extra types

Reply:
Right, the intention is just to minimize friction for users who might be aggregating errors.